### DISCUSSION TITLE
Subcommands not available anymore?

### DISCUSSION THREAD

**Discussion by mpern** [getsops/sops#1538] [Q&A] [answered]
Hi all!

I'm just starting out with sops and it looks like the documentation is not up-to-date anymore?

Subcommands like `encrypt`, `decrypt`, `edit` are not available and simply don't work.

e.g

```sh
sops encrypt --age age12345abcd  test.yaml
Failed to get the data key required to decrypt the SOPS file.
```

it looks like it tries to decrypt a file named `encrypt`, not encrypt `test.yaml`

**Comment by felixfontein** [comment, marked as answer]
The documentation in https://github.com/getsops/sops/ describes the state of the current `main` branch. The new subcommands have not yet appeared in any release. I hope that 3.9.0 will come out soon, that will contain them...

**Reply by mpern** [reply]
Ah, good to  know.

I was going by the published docs, btw

https://getsops.io/docs/

**Reply by felixfontein** [reply]
Ah. That page is very new, and not completely done yet (as you can notice when clicking on "Blog").

**Reply by felixfontein** [reply]
3.9.0 has just been released, so now subcommands are available!

**Reply by felixfontein** [reply]
FYI, getsops.io now contains the up-to-date docs for the latest released version, while this repo's README points to that page.
